api/Window.json

```diff
@@ -1981,10 +1981,10 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
```

Comment:
Looks like we should just remove `dialogArguments` from BCD. This change is correct though, it's not in Firefox :)